src/web/mjs/connectors/CarToonMad.mjs

```diff
@@ -0,0 +1,64 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class CarToonMad extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'cartoonmad';
+        super.label = 'CarToonMad';
+        this.tags = ['manga', 'chinese'];
+        this.url = 'https://www.cartoonmad.com';
+        this.requestOptions.headers.set('x-referer', this.url);
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        const request = new Request(new URL('/comic99.html', this.url), this.requestOptions);
+        const data = await this.fetchDOM(request, 'td[align] > a:last-child.pages');
+        const pageCount = parseInt(data[0].href.match(/(\d+)\.html/)[1]);
+        for(let page = 1; page <= pageCount; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangaList.push(...mangas);
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        const request = new Request(new URL('/comic99.'+('00' + page).slice(-2)+'.html', this.url), this.requestOptions);
```

Comment:
May use `padStart` instead of slice for readability
`String(page).padStart(2, '0')`